src/components/models/googleSign.js

```diff
@@ -182,7 +185,7 @@ const GoogleModel = ({
                         Are you sure you want to share this Project to Google Classroom?
                       </p>
                       */}
-                      <button type="submit">Confirm</button>
+                      {!loading &&<button type="submit">Confirm</button>}
```

Comment:
Do we remove the button at all? Not disabled?

Reply:
this is the requirement